index.html

```diff
@@ -0,0 +1,537 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Educatia</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="img/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="img/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="img/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/site.webmanifest" />
+    <script
+      src="https://kit.fontawesome.com/1ba545dca5.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body class="position-relative">
+    <header class="pl-2 p-sm-0">
+      <nav class="row d-none d-md-block">
+        <div class="col-sm p-2 d-flex justify-content-end col-gap bg-dark pr-3">
+          <a href="#" rel="noopener" class="text-white"
+            ><i class="fab fa-facebook"></i
+          ></a>
+          <a href="#" rel="noopener" class="text-white"
+            ><i class="fab fa-twitter"></i
+          ></a>
+          <a href="#" rel="noopener" class="text-white">English</a>
+          <a href="#" rel="noopener" class="text-white">My Page</a>
+          <a href="#" rel="noopener" class="text-white">Logout</a>
+        </div>
+      </nav>
+      <nav class="row">
+        <div class="col-sm-1"></div>
+        <nav class="col-sm-10 navbar navbar-expand-md p-1">
+          <div>
+            <button
+              class="navbar-toggler border-0 font-weight-bold mt-4"
+              type="button"
+              data-toggle="collapse"
+              data-target="#navbarTogglerMenu"
+              aria-controls="navbarTogglerMenu"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <i class="fas fa-bars h2 text-dark-50"></i>
+            </button>
+          </div>
+
+          <div class="collapse navbar-collapse bg-white" id="navbarTogglerMenu">
+            <a
+              class="navbar-brand p-0 d-none d-md-block"
+              href="index.html"
+              rel="noopener"
+            >
+              <img src="img/logo.PNG" alt="Logo" />
```

Comment:
- [ ] The logo link is breaking, please kindly fix that, see image below;
![kinglogo](https://user-images.githubusercontent.com/65852621/138378614-0cc3d98f-ccd5-4a42-af81-de530787afd2.png)